research/src/pages/breadcrumb.proposal.mdx

```diff
@@ -0,0 +1,66 @@
+---
+menu: Components
+name: Breadcrumb (Editor's Draft)
+path: /components/breadcrumb
+pathToResearch: /components/breadcrumb.research
+---
+
+import BreadcrumbAnatomy from '../components/breadcrumb-anatomy'
+
+## Overview
+
+Breadcrumbs are navigation land marks used to show a user's location within the site or app.
+
+## Use Cases
+
+A `<breadcrumb />` trail consists of a list of links to the parent pages of the current page in hierarchical order. It helps users find their place within a website or web application. Breadcrumbs are often placed horizontally before a page's main content.
+
+## Prior Art/Examples <a href="#prior-art" id="prior-art"></a>
+
+- [Ant Design](https://ant.design/components/breadcrumb/)
+- [Bootstrap](https://getbootstrap.com/docs/4.3/components/breadcrumb/)
+- [Auth0](https://styleguide.auth0.com/components/breadcrumb)
+- [Atlas Kit](https://atlassian.design/components/breadcrumbs/examples)
+- [Carbon](https://www.carbondesignsystem.com/components/breadcrumb/code/)
+- [Fabric](https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb)
+- [Lightning](https://www.lightningdesignsystem.com/components/breadcrumbs/)
+- [Semantic UI](https://semantic-ui.com/collections/breadcrumb.html)
+
+### Anatomy <a class="link" href="#anatomy" id="anatomy"></a>
+
+#### Diagram <a class="link" href="#diagram" id="diagram"></a>
+
+<BreadcrumbAnatomy />
+
+#### Structure <a class="link" href="#structure" id="structure"></a>
+
+- `<breadcrumb>` - The root element that that has the semantic value of navigation **[required]**
+- `<ordered-list>` - The list containing items and dividers **[required]**
+- `<list-item>` - The wrapper to items or divider content **[required]**
+
+Proposed HTML:
+
+```html
+<nav>
+  <ol>
+    <li><a>Item 1</a></li>
+    <li aria-hidden="true">/</li>
+    <li><a>Item 2</a></li>
+  </ol>
+</nav>
+```
+
+#### Accessibility <a class="link" href="#accessibility" id="accessibility"></a>
+
+The Breadcrumb is composed by:
+
+- A `nav` equivalent wrapper that should always have `aria-label` or `aria-labelledby`
+- A ordered `list`
+- `li` items and dividers where the `li` represeting the item can contain anchor or other elements and the `li` represeting the divider should have `aria-hidden="true"`
```

Comment:
A few notes here based on feedback in another thread:

1) Should we use the name divider or separator and why?
2) Does the li actually represent the item if it is not actionable or is the item slotted content?
3) If the expectation is that a separator (or divider) slotted in as part of a list item is presentational only - therefore, *always hidden*, why make that the rule? It seems more intuitive to expect that we have a presentational recommendation with this as "allowable". Either way, I think we need a caveat that this can be handled a couple of ways (per Charles' comment elsewhere).

Reply:
1. Doesn't look like there's any consensus between libs, but looks like a `space X tabs` discussion. I have no problem with either names. 
2. I believe it's the slot for the item as shown in the anatomy
3. The usage of `aria-hidden` was the common practice I found in the libs that are not using the divider as an pseudo element.

Reply:
1. Issue #200 created to discuss this
2. Leaving for now
3. Issue #201 created to discuss

Reply:
Updated this PR to reference the open discussions